data/remnant/remnant jobs.txt

```diff
@@ -150,7 +150,7 @@ mission "Remnant: Bounty 3"
 			names "korath"
 			cargo 3
 			variant
-				"Korath Dredger (Crippled)"
+				"Rano'erek (Crippled)"
```

Comment:
GitHub won't let me leave a suggestion there (line 133), but do you not want to change the `name` of this mission from `Damaged Dredger Bounty` to `Damaged Rano'erek Bounty`?

Reply:
Would that not cause issues with backwards compatability?

Reply:
No. The `name` of a mission is just the display name, it doesn't have to be unique.